mods/seasonal/moves.js

```diff
@@ -3482,7 +3489,7 @@ let BattleMovedex = {
 		accuracy: 100,
 		basePower: 90,
 		category: "Physical",
-		desc: "The user boosts its Speed by 1 stage and recovers half the HP lost by the target, rounded half up. This move has a 50% chance to boost the users Speed by 1 stage. If Big Root is held, the user recovers 1.3x the normal amount of HP, rounded half down.",
+		desc: "The user boosts its Speed by 1 stage and recovers half the HP lost by the target, rounded half up. This move has a 50% chance to boost the user's Speed by 1 stage. If Big Root is held, the user recovers 1.3x the normal amount of HP, rounded half down.",
```

Comment:
??????????????????????????????????????

Reply:
`The user boosts its Speed by 1 stage and recovers half the HP lost by the target, rounded half up. This move has a 50% chance to boost the user's Speed by 1 stage.`

Reply:
you dont need to tag me on discord and also comment on here.